src/DynamicSpriteFont.cs

```diff
@@ -144,11 +155,22 @@ public float DrawString(SpriteBatch batch, string text, Vector2 pos, Color color
 		}
 
 		public float DrawString(SpriteBatch batch, string text, Vector2 pos, Color[] glyphColors)
+		{
+			return DrawString(new SpriteBatchGlyphRenderer(batch), text, pos, glyphColors);
+		}
+		
+		public float DrawString(IGlyphRenderer batch, string text, Vector2 pos, Color[] glyphColors)
 		{
 			return DrawString(batch, text, pos, glyphColors, Vector2.One);
 		}
 
-		public float DrawString(SpriteBatch batch, string text, Vector2 pos, Color[] glyphColors, Vector2 scale, float depth = 0f)
+		public float DrawString(SpriteBatch batch, string text, Vector2 pos, Color[] glyphColors, Vector2 scale,
+			float depth = 0f)
+		{
+			return DrawString(new SpriteBatchGlyphRenderer(batch), text, pos, glyphColors, scale, depth);
```

Comment:
Can you, please, make SpriteBatchGlyphRender singleton with SpriteBatch settable property?
So above code would become like this:
```c#
  var renderer = SpriteBatchGlyphRenderer.Instance;
  renderer.SpriteBatch = batch;
  return DrawString(renderer, text, pos, glyphColors, scale, depth);
```
As it's bad practice to make an allocation every DrawString.